GettingStarted.md

```diff
@@ -129,7 +126,7 @@ data-min-scale           | 0         | Minimum scale factor.
 data-perspective         | 1000      | Perspective for 3D rendering. See https://developer.mozilla.org/en/CSS/perspective
 
 ### **Renaming Steps**
-You can give each step an ID. The name of the ID will be displayed in the browsers navigation bar instead of the default *step-x* whereas x is replaced by the current step number. This can be especially helpful, when trying to jump between steps and go back to a previous one. If you want to know how to move to a specific slide, you should take a look at the [DOCUMENTATION](DOCUMENTATION.md). This functionality is part of the "Goto" plugin
+You can give each step an ID. The name of the ID will be displayed in the browsers navigation bar instead of the default *step-x* whereas x is replaced by the current step number. This can be especially helpful, when trying to jump between steps and go back to a previous one. If you want to know how to move to a specific slide, you should take a look at the [DOCUMENTATION](./src/plugins/goto/README.md) of the "Goto" plugin. 
```

Comment:
Strictly speaking it's README, not DOCUMENTATION. Again, it's not a big deal semantically, but can be confusing to areader.

Reply:
Fair enough, will resolve